terraform/service_admin_apiserver.tf

```diff
@@ -74,7 +74,7 @@ resource "google_cloud_run_service" "adminapi" {
   template {
     spec {
       service_account_name = google_service_account.adminapi.email
-      timeout_seconds      = 25
+      timeout_seconds      = 10
```

Comment:
this needs to be higher due to bulk issue

Reply:
Done